core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -122,47 +124,59 @@ public Handle setStatementBuilder(StatementBuilder builder) {
      */
     @Override
     public void close() {
+        final List<Throwable> suppressed = new ArrayList<>();
         if (closed) {
             return;
         }
 
-        boolean wasInTransaction = isInTransaction()
-            && forceEndTransactions
-            && config.get().get(Handles.class).isForceEndTransactions();
+        boolean wasInTransaction = forceEndTransactions
+                && config.get().get(Handles.class).isForceEndTransactions();
+
```

Comment:
Why did the `isInTransaction()` part get removed?

Reply:
It didn't; it's simply moved down below.  I reorganized the `A() && B && C` clause to `if (B && C) { A() }` since two of them are local booleans and `isInTransaction` potentially calls out to the remote database server or throws.